src/pp-annotator/api/Storage.interface.ts

```diff
@@ -4,5 +4,5 @@ export default interface IStorage {
   create(annotation: IAnnotation): IAnnotation;
   update(annotation: IAnnotation): IAnnotation;
   delete(annotation: IAnnotation): IAnnotation;
-  query(id: number): IAnnotation;
+  query(query?: any): IAnnotation[];
```

Comment:
If I see correctly the interface is out of sync with the changes above

Reply:
Corrected